src/ui/RosterItem.ts

```diff
@@ -40,6 +40,10 @@ export default class RosterItem {
       });
 
       this.element.click(function() {
+         if ($(this).hasClass('jsxc-rosteritem-disabled'))
+         {
+            return;
+         }
```

Comment:
Is this really needed?

Reply:
i add jsxc-rosteritem-disabled class in BlockingCommandPlugin if a user was blocked... because the user should not open a chatbox with a blocked user. It simply make no sense to write to a blocked user, if this user is not able to answer anymore..

Reply:
I understand that, but you are hiding all disabled entries, so a user can't click that item.